### PR TITLE
Readd implicit package references section

### PR DESCRIPTION
**Pull request by gewarren**
For customers targeting older versions, we still need this info as it's linked from the product using https://aka.ms/sdkimplicitrefs. This info used to live in [csproj.md](https://github.com/dotnet/docs/pull/22277).

[Preview link](https://review.docs.microsoft.com/en-us/dotnet/core/project-sdk/overview?branch=pr-en-us-22314#implicit-package-references).

### CODE REVIEW

**Review by adegeo — APPROVED** [review]
Approved pending review from area expert.

**Comment by wli3** [comment]
@dsplaisted This looks good to me. But I think you should have better memory on the framework reference of 1.x 2.x time

File: docs/core/project-sdk/overview.md
Comment (dsplaisted) [inline docs/core/project-sdk/overview.md:131]: Technically it's through 2.2, not just 2.1.

**Review by dsplaisted — APPROVED** [review]

File: docs/core/project-sdk/overview.md
Comment (gewarren) [inline docs/core/project-sdk/overview.md:131]:
```suggestion
When targeting .NET Core 1.0 - 2.2 or .NET Standard 1.0 - 2.0, the .NET SDK adds implicit references to certain *metapackages*. A metapackage is a framework-based package that consists only of dependencies on other packages. Metapackages are implicitly referenced based on the target framework(s) specified in the [TargetFramework](msbuild-props.md#targetframework) or [TargetFrameworks](msbuild-props.md#targetframeworks) property of your project file.
```

**Comment by gewarren** [comment]
@wli3 @dsplaisted @adegeo Thanks for reviewing!

**Comment by gewarren** [comment]
I also updated the aka.ms link to point to https://docs.microsoft.com/dotnet/core/project-sdk/overview#implicit-package-references.

**Pull request merged**
